### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by Arau**
- Ubuntu 20.04 as default for Anthos on AWS
- Update default images in case that they are needed.

### CODE REVIEW

File: google-anthos-aws/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ What is included in this folder:
 On the admin host from which the Google Anthos k8s on AWS is accessible, run the following: 
 
 ```shell
-kubectl storageos install --include-etcd --etcd-cluster-yaml ondat-etcd.yaml --etcd-namespace storageos --stos-cluster-yaml ondat-cluster.yaml --stos-cluster-namespace storageos --stos-version v2.5.0-beta.1
+kubectl storageos install --include-etcd --etcd-cluster-yaml ondat-etcd.yaml --etcd-namespace storageos --stos-cluster-yaml ondat-cluster.yaml --stos-cluster-namespace storageos --stos-version v2.5.0
```
Comment (cvlc) [inline google-anthos-aws/README.md:18]:
```suggestion
kubectl storageos install --include-etcd --etcd-cluster-yaml ondat-etcd.yaml --etcd-namespace storageos --stos-cluster-yaml ondat-cluster.yaml --stos-cluster-namespace storageos --stos-version v2.6.0
```

File: google-anthos-aws/README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Let's have a look at the parameters:
 - ```--etcd-namespace storageos```: overwrite the default plugin parameters to deploy the etcd cluster within a custom namespace
 - ```--stos-cluster-yaml ondat-cluster.yaml```: overwrite the default plugin parameters with the one from this configuration file
 - ```--stos-cluster-namespace storageos```: overwrite the default plugin parameters to deploy the Ondat cluster within a custom namespace
-- ```--stos-version v2.5.0-beta.1```: overwrite the default plugin parameters with the desired version
+- ```--stos-version v2.5.0```: overwrite the default plugin parameters with the desired version
```
Comment (cvlc) [inline google-anthos-aws/README.md:27]:
```suggestion
- ```--stos-version v2.6.0```: overwrite the default plugin parameters with the desired version
```

File: google-anthos-aws/ondat-cluster.yaml [hunk 3]
```diff
@@ -131,15 +131,15 @@ spec:
   k8sDistro: "upstream"
   storageClassName: storageos
 #  images:
-#    nodeContainer: "storageos/node:v2.5.0-beta.1"
-#    apiManagerContainer: storageos/api-manager:v2.5.0-sync
+#    nodeContainer: "storageos/node:v2.5.0"
```
Comment (cvlc) [inline google-anthos-aws/ondat-cluster.yaml:134]:
```suggestion
#    nodeContainer: "storageos/node:v2.6.0"
```

**Comment by cvlc** [comment]
Added updates for `v2.6.0`

**Comment by cannischan** [comment]
@Arau please review the changes proposed by @cvlc - once it looks good just feel free to merge

**Comment by chris-milsted** [comment]
I am proposing we close this as these steps are now covered in the offline install guide and the --dry-run flag.

**Pull request closed without merging**